pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -120,113 +137,744 @@ func (r *ReconcileHyperConverged) Reconcile(request reconcile.Request) (reconcil
 		return reconcile.Result{}, err
 	}
 
+	// Add conditions if there are none
```

Comment:
I think `Readiness=false` is accurate always during initialization of the HCO, so I think we should it before the first reconcile occurs.   Maybe somewhere in `cmd/`

Reply:
Since the readiness probe is checking for that file the `ready` package creates. We could simply remove [these lines](https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/cmd/hyperconverged-cluster-operator/main.go#L78-L85) to have the behavior you specify.

Reply:
👍 Let's remove those lines.

Reply:
Do you agree/disagree that initialization should always be `Readiness=false`?

Reply:
I think it makes the most sense to start "unready" and not change that until you are "ready".